packages/preferences/src/browser/util/preference-tree-generator.ts

```diff
@@ -69,8 +69,13 @@ export class PreferenceTreeGenerator {
     });
 
     protected createLeafNode = (property: string, preferencesGroup: Preference.Branch): SelectableTreeNode => {
-        const propertySpecifier = this.split(property).slice(1);
-        const name = propertySpecifier.map(word => word.slice(0, 1).toLocaleUpperCase() + word.slice(1)).join(' ').trim();
+        const propertyWithoutCategory = property.split('.').slice(1);
```

Comment:
It might make it a little clearer what this array contains to do something like
```
const [_category, rawLeafName, rawLeafValue] = property.split('.');
```
and then derive the final `name` from manipulations of `rawLeafName` and `rawLeafValue` (if it exists).

Reply:
It does make the code clearer. Thank you for that.
Done